source/user-manual/reference/ossec-conf/index_manager.rst

```diff
@@ -21,27 +24,23 @@ Here is an example of the proper location of the *alerts* configuration section:
         </alerts>
     </wazuh_config>
 
-The ``agent.conf`` file is very similar to ``ossec.conf`` but ``agent.conf`` is used to centrally distribute configuration information to agents. See more :doc:`here <../centralized-configuration>`.
+The ``shared/agent.conf`` file is very similar to ``manager.conf`` but ``shared/agent.conf`` is used to centrally distribute configuration information to agents. See more :doc:`here <../centralized-configuration>`.
 
 Wazuh can be installed in two ways: as a manager by using the "server/manager" installation type and as an agent by using the "agent" installation type.
 
 +---------------------------------------------------------------------+------------------------+
 | Configuration sections                                              | Supported installations|
```

Comment:
I think the second column is not necessary.

Reply:
Done